### PR TITLE
Fix infinite-loading infinitely loading

### PR DESCRIPTION
**Pull request by roschaefer**
## 🍰 Pullrequest
The infinite loading component was indeed loading infinitely.

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: webapp/pages/profile/_id/_slug.vue [hunk 1]
```diff
@@ -238,8 +238,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-immediate-check="true"
```
Comment (mattwr18) [inline webapp/pages/profile/_id/_slug.vue]: what is the purpose of adding this?
doesn't this default to true anyways?
https://sroze.github.io/ngInfiniteScroll/documentation.html
or am I looking at a different library??

File: webapp/pages/profile/_id/_slug.vue [hunk 2]
```diff
@@ -238,8 +238,10 @@
...
+          :infinite-scroll-disabled="$apollo.loading"
+          :infinite-scroll-distance="10"
+          :infinite-scroll-throttle-delay="800"
```
Comment (mattwr18) [inline webapp/pages/profile/_id/_slug.vue]: default is 200ms... what went into the thinking to make it `800`? just your feeling? or was there something specific?

Comment (roschaefer) [reply webapp/pages/profile/_id/_slug.vue]: It was try/error with @alina-beck. We tested it with seed data. Maybe we can increase it even further after we learned about the low performance of our neo4j pod.

File: webapp/pages/profile/_id/_slug.vue (hunk 1 above)
Comment (roschaefer) [reply webapp/pages/profile/_id/_slug.vue]: Right! I double-checked and it seems it is the default per documentation. I wonder why this fixed some our cypress tests earlier. Maybe another change or the documentation is just wrong.

Comment (roschaefer) [reply webapp/pages/profile/_id/_slug.vue]: Done, ping @mattwr18

**Pull request merged**